crates/re_ui/src/list_item2/scope.rs

```diff
@@ -255,10 +276,11 @@ pub fn list_item_scope<R>(
         left_column_width,
         reserve_action_button_space: layout_stats.is_action_button_used,
         scope_id,
+        property_content_max_width: layout_stats.property_content_max_width,
     };
 
     // push, run, pop
-    LayoutInfoStack::push(ui.ctx(), state);
+    LayoutInfoStack::push(ui.ctx(), state.clone());
```

Comment:
why the new clone?

Reply:
Not sure. Not sure either why clippy doesn't complain...